lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java

```diff
@@ -71,7 +110,12 @@ public static int computeFactorial(int n) {
    * @return The sum of all the values.
    */
   public static double addNumbers(double[] values) {
-    return 0;
+    double sum = 0;
+    for (double num : values) {
+      sum += num;
+    }
+
+    return sum;
```

Comment:
You can make this a one line function using Stream.reduce().

Reply:
The Stream.reduce() function seems to only work with array lists/object lists

Reply:
I just figured out how to do this, I'm sure you can figure this out to. Give it a shot.